src/plugins/timeConductor/pluginSpec.js

```diff
@@ -126,3 +127,23 @@ describe('time conductor', () => {
     });
 
 });
+
+describe('duration functions', () => {
+    it('should transform milliseconds to DHMS', () => {
+        const functionResults = [millisecondsToDHMS(0), millisecondsToDHMS(86400000),
+            millisecondsToDHMS(129600000), millisecondsToDHMS(661824000)];
+        const validResults = [' ', '+ 1d', '+ 1d 12h', '+ 7d 15h 50m 24s'];
+        for (let i = 0; i < functionResults.length; i++) {
+            expect(validResults[i]).toEqual(functionResults[i]);
+        }
```

Comment:
I believe so we could be able to use directly `expect(validResults).toEqual(functionResults);` instead of the for loop statement.